vspace/src/main/java/edu/asu/diging/vspace/web/staff/ListImagesController.java

```diff
@@ -19,7 +20,7 @@
     private IImageService imageService;
 
     @RequestMapping("/staff/images/list/{page}")
-    public String listSpaces(@PathVariable String page, Model model) {
+    public String listSpaces(@PathVariable String page, @RequestParam(value = "sort", required = false) String sortedBy, @RequestParam(value = "order", required = false) String order, Model model) {
```

Comment:
since you're working on this, let's change the method name. that doesn't sound right.